bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoCoffeeHandler.java

```diff
@@ -90,23 +90,23 @@ public WemoCoffeeHandler(Thing thing, UpnpIOService upnpIOService, WemoHttpCall
         this.wemoCall = wemoHttpCaller;
         this.service = upnpIOService;
 
-        logger.debug("Creating a WemoCoffeeHandler V0.4 for thing '{}'", getThing().getUID());
+        logger.debug("Creating a WemoCoffeeHandler for thing '{}'", getThing().getUID());
     }
 
     @Override
     public void initialize() {
         Configuration configuration = getConfig();
-        host = (String) configuration.get("ipaddress");
+        host = (String) configuration.get(IPADDRESS);
 
-        if (configuration.get("udn") != null) {
+        if (configuration.get(UDN) != null) {
             logger.debug("Initializing WemoCoffeeHandler for UDN '{}'", configuration.get("udn"));
             service.registerParticipant(this);
 
             pollingJob = scheduler.scheduleWithFixedDelay(this::poll, 0, DEFAULT_REFRESH_INTERVALL_SECONDS,
                     TimeUnit.SECONDS);
             updateStatus(ThingStatus.ONLINE);
         } else {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "missing UDN");
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "config-status.error.missing-udn");
```

Comment:
Is it tested? I believe the text should be prefixed with "@text/", otherwise it will litterally set that text.

Reply:
Need to check, as I copied it from another binding. Will testd and correct if needed.

Reply:
corrected with leading @text